test/aircraft/AircraftCollection.spec.js

```diff
@@ -0,0 +1,79 @@
+import ava from 'ava';
+import AircraftCollection from '../../src/assets/scripts/client/aircraft/AircraftCollection';
+import { ARRIVAL_AIRCRAFT_MODEL_MOCK, DEPARTURE_AIRCRAFT_MODEL_MOCK } from './_mocks/aircraftMocks';
+
+ava('does not throw with valid parameters', (t) => {
+    t.notThrows(() => new AircraftCollection());
+});
+
+ava('.add() throws when passed invalid params', (t) => {
+    const collection = new AircraftCollection();
+
+    t.throws(() => collection.add(null));
+});
+
+ava('.add() increases the `#_items` length by 1', (t) => {
+    const collection = new AircraftCollection();
+
+    collection.add(ARRIVAL_AIRCRAFT_MODEL_MOCK);
+
+    t.true(collection.length === 1);
+});
+
+ava('.remove() throws when the itemToRemove does not exist in `#_items`', (t) => {
+    const collection = new AircraftCollection();
+
+    collection.add(DEPARTURE_AIRCRAFT_MODEL_MOCK);
+
+    t.throws(() => collection.remove(123456789));
+});
+
+ava('.remove() decreases the `#_items` length by 1', (t) => {
+    const collection = new AircraftCollection();
+
+    collection.add(DEPARTURE_AIRCRAFT_MODEL_MOCK);
+
+    t.true(collection.length === 1);
+
+    collection.remove(DEPARTURE_AIRCRAFT_MODEL_MOCK);
+
+    t.true(collection.length === 0);
+});
+
+ava('.findAircraftByCallsign() returns null when no callsign is passed', (t) => {
+    const collection = new AircraftCollection();
+    const result = collection.findAircraftByCallsign();
+
+    t.falsy(result);
```

Comment:
honestly not a fan of the `falsy` matcher.  in this case I'd prefer:
```js
t.true(typeof result === 'undefined');
```

remember tests are another form of documentation.  I'm already a fan of verbosity, be even more so with tests for that reason.

Reply:
done